tests/App.test.jsx

```diff
@@ -0,0 +1,12 @@
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { shallow } from 'enzyme';
+import React from 'react';
+import App from '../client/src/component/App.jsx';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('review test', () => {
+  test('should render with no error', () => {
```

Comment:
Should be more specific in your test message? (i.e. "App should render/mount with no error")